src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -169,7 +169,6 @@ public function setup_meta( $meta ) {
 		}
 
 		// `source` will be empty when importing .ics files
-		$original_source = $this->meta['source'];
```

Comment:
We need (/want) this variable for the `tribe_aggregator_meta_source` hook, below.

Is the problem that source may not be set? If so, can we move this down rather than removing it completely?